dev/com.ibm.websphere.appserver.features/visibility/public/jpa-2.1/resources/l10n/com.ibm.websphere.appserver.jpa-2.1.properties

```diff
@@ -14,6 +14,6 @@
 #NLS_MESSAGEFORMAT_NONE
 #
 
-description=This feature enables support for applications that use application-managed and container\
--managed JPA written to the Java Persistence API 2.1 specification. The support is built on top of \
- EclipseLink to support the container-managed programming model.
+description=This feature enables support for applications that use application-managed \
+and container-managed JPA. This includes Java Persistence API 2.1 specification interfaces, \
+container-managed JPA integration, and EclipseLink as the default JPA provider implementation.
```

Comment:
**change**
`This includes Java Persistence API 2.1 specification interfaces, \
container-managed JPA integration, and EclipseLink as the default JPA provider implementation.`
**to**
`This support includes Java Persistence API 2.1 specification interfaces and 
container-managed JPA integration. EclipseLink is also supported, as the default JPA provider implementation.`